addins/Cake.MiniCover.yml

```diff
@@ -1,11 +1,11 @@
-
 Name: Cake.MiniCover
 NuGet: Cake.MiniCover
 Assemblies:
 - "/**/Cake.MiniCover.dll"
 Repository: https://github.com/nlowe/Cake.MiniCover
 Author: Nathan Lowe
 Description: "Cake addin for MiniCover"
+Prerelease: "true"
```

Comment:
@Jericho Wonder if this is correct to switch to pre-release just because an newer pre-release exists?

Reply:
As mentioned previously, I was in favor of ignoring pre-release addins but @gep13 thought otherwise.